app/assets/stylesheets/mobile/header.scss

```diff
@@ -2,56 +2,75 @@
 
 $drawer-width: 400px;
 $drawer-width-offset: $drawer-width + 50px;
+$mobile-navbar-height: 46px;
 
-header {
-  position: fixed;
```

Comment:
Please keep the header fixed at the top of the screen. It now disappears when we scroll.